pom.xml

```diff
@@ -159,6 +159,9 @@
                         <goals>
                             <goal>jar</goal>
                         </goals>
+                        <configuration>
+                            <doclint>all,-syntax</doclint>
```

Comment:
@Malax this following gets the build to pass on Java 11 runs but is really just covering up the underlying issue.  

from the [debug output i got the javadoc plugin to spit out](https://github.com/forcedotcom/sf-fx-sdk-java/actions/runs/3388506081/jobs/5630569985) (see the `cat` commands after the failure) there was no reason i could find that would explain why it would want to treat the `@ParametersAreNonnullByDefault` annotation in `package-info.java` as a javadoc tag.  i was not able to reproduce the error locally either.  

what's your take on this?

Reply:
Interesting. I think we should get to the bottom of this - if it happens during release too we will be unable to release. Adding `-syntax`, we might mask other errors we'd care about. I can take a closer look tomorrow.